docs/content/3.providers/imagekit.md

```diff
@@ -140,11 +140,11 @@ Specify background color and its opacity for your image using the `bg` modifier.
 > Read more about ImageKit crop, resize, and other common transformations [here](https://docs.imagekit.io/features/image-transformations/resize-crop-and-other-transformations).
 
 ## Overlay Transformation Modifiers
-Using ImageKit's Nuxt Image integration, you can overlay images or text over other images for watermarking or creating a dynamic banner using custom text!
+ImageKit's Nuxt Image integration enables you to apply overlays to [images](https://docs.imagekit.io/features/image-transformations/overlay-using-layers.md) and [videos](https://docs.imagekit.io/features/video-transformation/overlay.md) using the raw parameter with the concept of [layers](https://docs.imagekit.io/features/image-transformations/overlay-using-layers.md#layers). The raw parameter facilitates incorporating transformations directly in the URL. A layer is a distinct type of transformation that allows you to define an asset to serve as an overlay, along with its positioning and additional transformations.
```

Comment:
This library is for images only. Nuxt image integration does not allow for video overlays, does it?

Reply:
"A layer is a distinct type of transformation that allows you to define an asset to serve as an overlay, along with its positioning and additional transformations."

Instead of writing this, we can keep a simple sentence which conveys that you can apply text, image overlay and other transformation parameters on them.

Reply:
done